Datastore/src/EntityMapper.php

```diff
@@ -57,12 +65,19 @@ class EntityMapper
      * @param bool $returnInt64AsObject If true, 64 bit integers will be
      *        returned as a {@see \Google\Cloud\Core\Int64} object for 32 bit
      *        platform compatibility.
+     * @param string $connectionType [optional] The connection type of the client.
+     *        Can be `rest` or `grpc`, defaults to `grpc`.
      */
-    public function __construct($projectId, $encode, $returnInt64AsObject)
-    {
+    public function __construct(
+        $projectId,
+        $encode,
+        $returnInt64AsObject,
+        $connectionType = 'grpc'
```

Comment:
Perhaps we would want to mark this as `@internal` (if this is not exposed to the users), so that we don't have a problem later when we remove the connection classes.